content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -41,7 +41,7 @@ When systemd is chosen as the init system for a Linux distribution, the init pro
 and consumes a root control group (`cgroup`) and acts as a cgroup manager. Systemd has a tight
 integration with cgroups and will allocate cgroups per process. It's possible to configure your
 container runtime and the kubelet to use `cgroupfs`. Using `cgroupfs` alongside systemd means
-that there will then be two different cgroup managers.
```

Comment:
/LGTM

Reply:
The original is also idiomatic english, but there's nothing wrong with this revision.